<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuizShow</class>
 <widget class="QWidget" name="QuizShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>70</red>
        <green>82</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>70</red>
        <green>82</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>70</red>
        <green>82</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>70</red>
        <green>82</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>70</red>
        <green>82</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>70</red>
        <green>82</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>70</red>
        <green>82</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>70</red>
        <green>82</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>70</red>
        <green>82</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>QuizShow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(70, 82, 255);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>81</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="button_resign">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url();</string>
       </property>
       <property name="text">
        <string>R</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_fifty_fifty">
       <property name="styleSheet">
        <string notr="true">background-image: url();</string>
       </property>
       <property name="text">
        <string>1/2</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_phone">
       <property name="styleSheet">
        <string notr="true">background-image: url();</string>
       </property>
       <property name="text">
        <string>T</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_public">
       <property name="styleSheet">
        <string notr="true">background-image: url();</string>
       </property>
       <property name="text">
        <string>P</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="button_next">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 128);
color: rgb(255, 250, 249);
background-image: url();</string>
    </property>
    <property name="text">
     <string>Kliknij aby rozpocząć grę</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>181</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 250, 249);
background-color: rgb(0, 0, 128);
background-image: url();</string>
    </property>
    <property name="autoScrollMargin">
     <number>14</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="currentRow">
     <number>14</number>
    </property>
    <item>
     <property name="text">
      <string>15    1 MILION</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14    500000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13    250000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12    125000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11    64000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10    32000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> 9     16000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> 8     8000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> 7     4000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> 6     2000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> 5     1000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> 4     500</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> 3     300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> 2     200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string> 1     100</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelQ">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>741</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 128);
color: rgb(255, 250, 249);
background-image: url();
border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>Treść pytania</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>741</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="button_answA">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>332</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>332</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 128);
background-image: url();
color: rgb(255, 250, 249);
border-radius: 10px;</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="text">
          <string>odpA</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_answB">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>332</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>332</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 128);
color: rgb(255, 250, 249);
border-radius: 10px;
background-image: url();</string>
         </property>
         <property name="text">
          <string>odpB</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="button_answC">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>332</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>332</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(0, 0, 128);
color: rgb(255, 250, 249);
border-radius: 10px;
background-image: url();</string>
         </property>
         <property name="text">
          <string>odpC</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_answD">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>332</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>332</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(0, 0, 128);
color: rgb(255, 250, 249);
border-radius: 10px;
background-image: url();</string>
         </property>
         <property name="text">
          <string>odpD</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_test">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>421</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url();</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
